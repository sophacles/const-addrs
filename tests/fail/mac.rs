use const_addrs::mac;

fn main() {
    let _ = mac!("ca:fe:ca:fe");
    let _ = mac!("ca:fe:ca:fe:ca:fe:cafe");
}
