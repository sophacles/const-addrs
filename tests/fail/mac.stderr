error: Invalid length of 11 characters
 --> tests/fail/mac.rs:4:18
  |
4 |     let _ = mac!("ca:fe:ca:fe");
  |                  ^^^^^^^^^^^^^
